plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -65,7 +64,7 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
 		// this will cause a live lookup of the namespace to get its latest state even
 		// before the watch notification is received.
 		if a.GetOperation() == admission.Delete {
-			l.store.Delete(&api.Namespace{
+			l.informer.GetStore().Delete(&api.Namespace{
```

Comment:
No.  This can't be done.  This will mutate the cache for everyone.  This can't share an informer yet.  You'll need a followup issue to create an overlay on the cache.